pr_agent/algo/ai_handler.py

```diff
@@ -99,28 +109,39 @@ async def chat_completion(self, model: str, system: str, user: str, temperature:
                 )
             if self.azure:
                 model = 'azure/' + model
+            system = get_settings().get("CONFIG.MODEL_SYSTEM_PREFIX", "") + system + \
+                     get_settings().get("CONFIG.MODEL_SYSTEM_SUFFIX", "")
+            user = get_settings().get("CONFIG.MODEL_USER_PREFIX", "") + user + \
+                   get_settings().get("CONFIG.MODEL_USER_SUFFIX", "")
             messages = [{"role": "system", "content": system}, {"role": "user", "content": user}]
```

Comment:
**Suggestion:** The `chat_completion` method is becoming quite large and complex. Consider breaking it down into smaller, more manageable functions. For example, the code that constructs the `messages` list could be moved to a separate method. This will improve readability and maintainability. [maintainability]
```suggestion
            def construct_messages(self, system: str, user: str):
                system = get_settings().get("CONFIG.MODEL_SYSTEM_PREFIX", "") + system + \
                         get_settings().get("CONFIG.MODEL_SYSTEM_SUFFIX", "")
                user = get_settings().get("CONFIG.MODEL_USER_PREFIX", "") + user + \
                       get_settings().get("CONFIG.MODEL_USER_SUFFIX", "")
                return [{"role": "system", "content": system}, {"role": "user", "content": user}]

            messages = self.construct_messages(system, user)
```